sympy/solvers/solvers.py

```diff
@@ -2453,24 +2481,33 @@ def solve_linear_system(system, *symbols, **flags):
 
 
 def solve_undetermined_coeffs(equ, coeffs, sym, **flags):
-    """Solve equation of a type p(x; a_1, ..., a_k) == q(x) where both
-       p, q are univariate polynomials and f depends on k parameters.
-       The result of this functions is a dictionary with symbolic
-       values of those parameters with respect to coefficients in q.
+    """
+    Solve equation of a type ``p(x; a_1, ..., a_k) == q(x)`` where both
+    ``p``, ``q`` are univariate polynomials and ``f`` depends on ``k``
```

Comment:
Minor grammar cleanup for this sentence: Solve **an** equation of type ... where both p **and** q are ...

Reply:
I don't know what `f` is referring to here.

Reply:
I would just rewrite this as "p and q are univariate polynomials that depend on k parameters"